app/models/miq_request_workflow.rb

```diff
@@ -848,7 +848,7 @@ def load_ems_node(item, log_header)
                  else
                    item.class.base_class
                  end
-    node = @ems_xml_nodes[klass_name][item.id]
+    node = @ems_xml_nodes.dig("klass_name", "item.id")
```

Comment:
Not sure I like the `.dig`, as a `.fetch(klass_name, {})[item.id]` would also do the trick.

Also, pretty sure you don't want to be quoting things here, but if you are (and there is some weird `.send` magic involved with `.dig`), then that is even more of a reason to use the above.